src/dnx.clr.managed/RuntimeBootstrapper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace dnx.clr.managed
+{
+    internal class RuntimeBootstrapper
+    {
+        public static int Execute(string[] argv)
+        {
+            var executeMethodInfo = GetBootstrapperType()
+                .GetMethod("Execute", BindingFlags.Static | BindingFlags.Public, null, new[] { argv.GetType() }, null);
+            return (int)executeMethodInfo.Invoke(null, new object[] { argv });
+        }
+
+        public static Task<int> ExecuteAsync(string[] argv)
+        {
+            var executeMethodInfo = GetBootstrapperType()
+                .GetMethod("ExecuteAsync", BindingFlags.Static | BindingFlags.Public, null, new[] { argv.GetType() }, null);
+            return Task.FromResult((int)executeMethodInfo.Invoke(null, new object[] { argv }));
```

Comment:
This doesn't look correct. The method returns a Task<int> already, why not just cast?